### PR TITLE
Add subscription activation/deactivation resources

### PR DESCRIPTION
**Pull request by lounjukk**
Add user subscription activation and deactivation resources to UserService.

Closes: https://github.com/lensapp/lens-platform-sdk/issues/269
TODO: Integration tests

### CODE REVIEW

File: src/UserService.ts [hunk 1]
```diff
@@ -132,6 +134,67 @@ class UserService extends Base {
...
+          if (typeof message === "string") {
+            if (message.includes("User")) {
+              return new UserNameNotFoundException(username);
+            }
+
+            return new NotFoundException(`Recurly subscription ${license.subscriptionId} not found`);
+          }
+
+          return new LensSDKException(
+            500,
```
Comment (panuhorsmalahti) [inline src/UserService.ts:156]: We should use the correct error code to make debugging easier.

File: src/types/types.ts [hunk 2]
```diff
@@ -12,3 +12,14 @@ export interface EntityType {
 }
 
 export type MapToEntity<Type extends EntityType> = MapPropsToDate<Type, "createdAt" | "updatedAt">;
+
+export enum LicenseType {
```
Comment (panuhorsmalahti) [inline src/types/types.ts:16]: I'd prefer avoiding `enum`.

Comment (DmitriyNoa) [reply src/types/types.ts]: Moved from enum to the same as used in the API

File: src/UserService.ts [hunk 3]
```diff
@@ -132,6 +134,67 @@ class UserService extends Base {
...
```
[the remark this answers is not in the document]
Comment (DmitriyNoa) [reply src/UserService.ts]: Added required statuses

File: src/UserService.ts (hunk 1 above)
Comment (DmitriyNoa) [reply src/UserService.ts]: Added required statuses

**Review by panuhorsmalahti — APPROVED** [review]

**Pull request merged**